packages/apps/graph/src/services/chainweb-node/mempool.ts

```diff
@@ -0,0 +1,59 @@
+import { chainIds } from '@utils/chains';
+
+export class MempoolError extends Error {
+  public mempoolError: any;
+
+  constructor(message: string, mempoolError?: any) {
+    super(message);
+    this.mempoolError = mempoolError;
+  }
+}
+
+export async function mempoolGetPending() {
+  try {
+    process.env.NODE_TLS_REJECT_UNAUTHORIZED = '0';
```

Comment:
This feels a bit hacky. Is this only an issue when talking to localhost? Also changing an nev var means it will have an effect on the whole environment as well